WORKSPACE

```diff
@@ -0,0 +1,61 @@
+# SPDX-FileCopyrightText: Copyright (c) 2023 by Rivos Inc.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+#
+# Rivos salus (hypervisor) workspace
+#
+
+workspace(name = "salus")
+
+#
+# Rivos rules, repositories, and toolchains
+#
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
+
+git_repository(
+    name = "rules_rivos",
+    commit = "cbc17fdf309e9edf353bdaa665d3baf03af1227f",
+    remote = "git@gitlab.ba.rivosinc.com:rv/sw/int/bazel_rules_rivos.git",
+    shallow_since = "1671543842 -0800",
+)
```

Comment:
This is going to break the build for CI and anyone outside of Rivos' VPN. Do we have an alternative?

Reply:
Working on getting this moved to github

Reply:
fixed